src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public Bulldozer(String name) {
+        setName(name);
```

Comment:
краще створити конструктор в батьківському класі і викликати його:
```suggestion
        super(name);
```